src/views/index.ejs

```diff
@@ -0,0 +1,407 @@
+            <a target="_blank" class="btn btn trail-btn focus-reset" href="https://finestdevs.com/shade/">
+              Join Now!
+            </a>
+          </div>
+          <!-- mobile menu trigger -->
+          <div class="mobile-menu-trigger">
+            <span></span>
+          </div>
+          <!--/.Mobile Menu Hamburger Ends-->
+        </nav>
+      </div>
+    </header>
+    <!-- navbar- -->
+    <!-- Hero Area -->
+    <div class="hero-area-l-14 position-relative z-index-1 overflow-hidden">
+      <div class="container">
+        <div class="row position-relative justify-content-center">
+          <div class="col-xl-5 col-lg-6 col-md-8 col-sm-10 pr-0 " data-aos="fade-right" data-aos-duration="800" data-aos-once="true">
+            <div class="content">
+              <h1>Dreams Will Come True</h1>
+              <p>If you are a dreamer who pursue, you can find supporters for your dreams with Shades. 
+                If you are a supporter, meet with the dreamers and help their dreams to come true. </p>
+              <a href="#" class="btn focus-reset">Join Now!</a>
+            </div>
+          </div>
+          <div class="col-xl-7 col-lg-5 col-md-8 " data-aos="fade-left" data-aos-duration="800" data-aos-once="true">
+            <div class="banner-image-l-14">
+              <img src="./image/l4/hero-img.png" alt="" class="w-100 mt-xl-n10">
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="bg-shape-14"></div>
+    </div>
+    <!-- Brand-area -->
+    <div class="brand-area-l-14">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-12 text-center">
+            <h4>Our Supporters and Sponsors</h4>
+          </div>
+        </div>
+        <div class="row img-grayscale">
+          <div class="col-lg-12">
+            <div class="brand-area-l-14-items d-flex justify-content-center justify-content-xl-between align-items-center flex-wrap ">
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="500" data-aos-once="true">
+                <img src="image/l4/brand-logo.svg" alt="">
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="700" data-aos-once="true">
+                <img src="image/l4/brand-logo-1.svg" alt="">
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="900" data-aos-once="true">
+                <img src="image/l4/brand-logo-2.svg" alt="">
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="1000" data-aos-once="true">
+                <img src="image/l4/brand-logo-3.svg" alt="">
+              </div>
+              <div class="single-brand " data-aos="fade-right" data-aos-duration="1200" data-aos-once="true">
+                <img src="image/l4/brand-logo-4.svg" alt="">
+              </div>
+            </div>
+          </div>
+        </div>

```

Comment:
- here we need to duplicate this section and convert it to technologies used in our api and insert these tech logos like nodejs, express,mongodb etc

![image](https://user-images.githubusercontent.com/14183741/152679217-71279ac8-9ee3-4344-9a07-e8497afb80e6.png)

Reply:
Done ✔️